src/cmds/query.ts

```diff
@@ -23,11 +23,16 @@ export const builder = {
     describe: 'Run all operations in order',
     type: 'boolean',
   },
+  variables: {
+    alias: 'g',
```

Comment:
`v` was taken so I took a random letter here.

Reply:
I think it's best to leave it out completely. We do that in other places too.

Reply:
I would take the alias out completely if it doesn't make sense.